llama-index-integrations/vector_stores/llama-index-vector-stores-azureaisearch/llama_index/vector_stores/azureaisearch/base.py

```diff
@@ -45,10 +46,27 @@ class IndexManagement(int, enum.Enum):
     CREATE_IF_NOT_EXISTS = auto()
 
 
-class AzureAISearchVectorStore(VectorStore):
+class AzureAISearchVectorStore(BasePydanticVectorStore):
     stores_text: bool = True
     flat_metadata: bool = True
 
+    from azure.search.documents.indexes import SearchIndexClient
+    from azure.search.documents import SearchClient
```

Comment:
These can just be imports at the top level I think

Reply:
ready!